src/PlusDataCollection/OptiTrack/vtkPlusOptiTrack.cxx

```diff
@@ -50,10 +50,14 @@ class vtkPlusOptiTrack::vtkInternal
   // Maps rigid body names to transform names
   std::map<int, PlusTransformName> MapRBNameToTransform;
 
-
   // Flag to run Motive in background if user doesn't need GUI
   bool RunMotiveInBackground;
 
+  // Time of last tool update
+  double LastToolUpdateTime;
```

Comment:
ToolUpdate is such broadly used terms that it has almost no meaning. It would be better to use a bit more specific name - see also https://github.com/PlusToolkit/PlusLibData/pull/7/files

Reply:
For example "MotiveDataDescriptionsUpdateTime"